utils/versions/build.gradle.kts

```diff
@@ -0,0 +1,56 @@
+import com.squareup.javapoet.FieldSpec
+import com.squareup.javapoet.JavaFile
+import com.squareup.javapoet.TypeName
+import com.squareup.javapoet.TypeSpec
+import javax.lang.model.element.Modifier.*
```

Comment:
no asterisk imports, this might help: https://github.com/sovity/customer-portal/blob/6cf9736bfd602dd80fb403377d2a95fe5e8f63ce/backend/.editorconfig#L15-L20

Reply:
yes that helped.